lib/stat_tracker.rb

```diff
@@ -178,4 +178,40 @@ def worst_coach(season_id)
 
         head_coach_win_percentages.min_by {|coach, win_percent| win_percent}.first
     end
+
+    def name_team_list
+        @teams.each_with_object(Hash.new) {|team, team_list| team_list[team.team_id] = team.team_name}
+    end
+
+    def most_accurate_team(season_id)
+        all_season_game_id = @games.map do |game|
+            game.game_id if game.season == season_id
+        end.compact
```

Comment:
Since it's a longer method, we may potentially want to add a space after the do's ends to help improve readability